Tests/Common/Securities/BrokerageModelSecurityInitializerTests.cs

```diff
@@ -119,10 +118,10 @@ public void BrokerageModelSecurityInitializer_CanSetPrice_ForQuoteBar()
             _algo.SetDateTime(dateForWhichDataExist);
 
             // Act
-            _brokerageInitializer.Initialize(_quoteBarSecurity, true);
+            _brokerageInitializer.Initialize(_quoteBarSecurity);
 
             // Assert
-            Assert.IsFalse(_quoteBarSecurity.Price == 0);
+            Assert.IsTrue(_quoteBarSecurity.Price == 0);
```

Comment:
I would either comment why this is expected to be true or remove the entire test method.

Arguably tests pertaining to setting a security's price don't belong in this test class, but rather would belong in tests or the seeder.

Not recommending, but if we wanted to confirm that the seeder either is or is not invoked, typically a fake or stub implementation would be used to achieve this. This way the test doesn't depend on an implementation of the seeder, but just the seeder contract (interface).